src/components/atoms/CyButton.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <button :class="`cy-button cy-button--${kind}`">
+    <div class="_button-text">
+      <img
+        src="cy-icon-black-transparent.svg"
+        width="16"
+        style="vertical-align: middle; margin-right: 8px; margin-bottom: 2px"
+      />{{ buttonLabel }}
+    </div>
+  </button>
+</template>
+
+<style lang="sass" scoped>
+// $
+._button-text
+  +t-h5()
+  margin-top: 10px
+  margin-bottom: 10px
+  color: black
+  text-transform: uppercase
+
+.cy-button
+  border: solid
+  border-color: black
+  border-width: 1px
+  background-color: white
+  padding: 6px 16px
```

Comment:
please change
```sass
padding: 6px 16px
```
with 
```sass
+py(6px)
+px($md)
```

### The reason:

readability ( because personally I'm used to using those mixins 😄  )
- py stands for "padding on the Y axis" (so vertical padding)
- px stands for "padding on the X axis" (so horizontal padding)

If you look at the `margin-padding.sass` file you'll see that there are some sizes based on 16px provided as variables:
<img width="274" alt="image" src="https://user-images.githubusercontent.com/3253920/94398492-97e9a900-01a0-11eb-886e-1bafb5b51cab.png">

And whenever using margin or padding I try to stick to those sizes as much as possible.
So whenever you need to write `16px` you can write `$md` instead. 🙏